pkg/auth/token/token.go

```diff
@@ -53,15 +53,15 @@ type Token struct {
 	Token     string
 }
 
-func (j *jwtManagerImpl) createToken(ctx context.Context, tenantId, name string, id *string) (*Token, error) {
+func (j *jwtManagerImpl) createToken(ctx context.Context, tenantId, name string, id *string, _expiresAt *time.Time) (*Token, error) {
```

Comment:
underscore is weird, please rename to `expAt`, `expires`, `initExpires`, or rename the other variable to `actualExpiresAt` or `newExpiresAt` etc.